File: data/imported/knowledge/slides/net/merged/snippets/snippet_076.cs
[Fact]
    public void TestRemoveAt()
    {
        // remove_at removes by index.
        using var pres = new Presentation();
        var slide = BlankSlide(pres);
        slide.Shapes!.AddAutoShape(ShapeType.Rectangle, 50, 50, 200, 100);
        slide.Shapes!.AddAutoShape(ShapeType.Ellipse, 300, 50, 150, 150);
        slide.Shapes!.RemoveAt(0);
        slide.Shapes!.Count.Should().Be(1);
    }

File: data/imported/knowledge/slides/java/merged/snippets/snippet_076.java
@Test
    void testMultipleShapeTypes() {
        ShapeType[] types = {ShapeType.RECTANGLE, ShapeType.ELLIPSE, ShapeType.TRIANGLE};
        try (var pres = new Presentation()) {
            var slide = blankSlide(pres);
            for (ShapeType st : types) {
                var s = slide.getShapes().addAutoShape(st, 10, 10, 100, 100);
                assertThat(s.getShapeType()).isEqualTo(st);
            }
            assertThat(slide.getShapes().size()).isEqualTo(3);
        }
    }